mito-ai/src/Extensions/AiChat/ChatMessage/ChatInput.tsx

```diff
@@ -146,6 +146,31 @@ const ChatInput: React.FC<ChatInputProps> = ({
         }
     };
 
+    const handlePaste = async (e: React.ClipboardEvent<HTMLTextAreaElement>): Promise<void> => {
```

Comment:
If I paste one image in and then paste another image in, the AI knows that it is receiving two images, but it thinks they are the same image. Only the first one is seen by the AI. 

Unlike if I drag and drop two images in, then both get sent